releasenotes/notes/fix-aux-ops-evaluator-83ce1606d1ad19b3.yaml

```
@@ -0,0 +1,6 @@
+fixes:
+  - |
+    The eval_observables() raised an error when the quantum_circuit is a 
+    StateFn. This comes from the fact that creating a StateFn from a StateFn
+    results in an OperatorStateFn. A check was added to avoid this error.
+    A test case was also added to assert that the error was indeed fixed.
```

The target audience for these notes are users of Terra, not mainly the developers, so we don't have to include technical details of how we fixed this 🙂 
```suggestion
    The :fun:`~qiskit.algorithms.eval_observables` function raised an error when the 
    ``quantum_state`` argument is a :class:`~qiskit.opflow.StateFn`. 
    This error was fixed and ``eval_observables`` now correctly supports all input types
    specifying in the type hints.
```